nginx.conf

```diff
@@ -0,0 +1,9 @@
+server {
+ listen 80;
+
+ location / {
+   root /usr/share/nginx/html/;
+   include /etc/nginx/mime.types;
+   try_files $uri $uri/ /index.html;
```

Comment:
I hope you understand in depth what is going on here.

Reply:
have an idea but definitely not in depth...will look into it..